core/templates/pages/exploration-editor-page/feedback-tab/thread-table/thread-table.component.ts

```diff
@@ -28,14 +28,18 @@ import { ThreadStatusDisplayService } from '../services/thread-status-display.se
   templateUrl: './thread-table.component.html'
 })
 export class ThreadTableComponent {
-  @Output('onClickRow') clickRowEventEmitter: EventEmitter<string> =
-  new EventEmitter();
+  @Output() clickRow: EventEmitter<string> =
+  (new EventEmitter());
```

Comment:
Can you change this to `rowClick` instead of `clickRow`
```suggestion
  @Output() rowClick: EventEmitter<string> = (
    new EventEmitter());
```

Reply:
Done